Services/Request/CategoryManager.php

```diff
@@ -19,7 +19,11 @@ public function __construct(Resolver $resolver, RequestInterface $request)
 
     public function getCategory()
     {
-        return $this->resolver->get()->getCurrentCategory();
+        try {
+            return $this->resolver->get()->getCurrentCategory();
+        } catch (\Exception $exception) {
+            return null;
```

Comment:
Because we're returning null modify Line 33 to add following,

``` 
$category &&
```

And Line 48,

``` 
if (!$category) {
  return '/';
}
```